h/static/scripts/group-forms/components/GroupModeration.tsx

```diff
@@ -1,11 +1,70 @@
+import { Spinner } from '@hypothesis/frontend-shared';
 import { useState } from 'preact/hooks';
 
 import type { Group } from '../config';
+import { useGroupAnnotations } from '../hooks/use-group-annotations';
+import type { APIAnnotationData } from '../utils/api';
 import GroupFormHeader from './GroupFormHeader';
 import type { ModerationStatus } from './ModerationStatusSelect';
 import ModerationStatusSelect from './ModerationStatusSelect';
 import FormContainer from './forms/FormContainer';
 
+type AnnotationsListProps = {
+  filterStatus?: ModerationStatus;
+};
+
+function AnnotationsList({ filterStatus }: AnnotationsListProps) {
```

Comment:
```suggestion
function AnnotationList({ filterStatus }: AnnotationsListProps) {
```

Should be `ThingList` rather than `ThingsList`. See eg. `ThreadList`, `GroupList` in the client.